bats/fb-test-foreman-ansible.bats

```diff
@@ -19,13 +19,15 @@ setup() {
   echo "- command: uptime" > "/etc/ansible/roles/${ROLE_NAME}/tasks/main.yml"
 
   # check if smart_proxy_ansible finds it and can import it
-  hammer ansible roles fetch --proxy-id 1 | grep "${ROLE_NAME}"
+  if ! hammer --output csv --no-headers ansible roles list --search="name=${ROLE_NAME}" | grep "${ROLE_NAME}"; then
+    hammer ansible roles fetch --proxy-id 1 | grep "${ROLE_NAME}"
+  fi
```

Comment:
Wouldn't it be easier to just unconditionally fetch without a grep? That way we test that we can always fetch. It just doesn't have to result in importing anything.

Reply:
That would also work, yeah